ekncontent/ekncontent/eknc-domain.c

```diff
@@ -39,7 +40,10 @@ struct _EkncDomain
   gchar *path;
   GList *subscriptions;
 
-  EkncXapianBridge *xapian_bridge;
+  char *language;
```

Comment:
Left over from debugging?

Reply:
Not really; we pass the language from the engine to the domain at construction time because the domain has to know how to build the query parser — which implies known how to create a stemmer for the given language.

Reply:
Never mind! I read the diff wrong and thought this was a global variable.